tests/types/src/valid.ts

```diff
@@ -738,6 +742,165 @@ const retrievedCartElement: StripeCartElement | null = elements.getElement(
   'cart'
 );
 
+const payButtonElementDefault = elements.create('payButton');
+
+const payButtonElement = elements.create('payButton', {
+  buttonHeight: 60,
```

Comment:
Could we change this to something like `55`? Our validators validate its a max of 55 (and while that doesn't matter here, it might be confusing to some who happen to read our tests)